alpha/apps/kaltura/lib/myEntryUtils.class.php

```diff
@@ -1742,11 +1742,14 @@ public static function copyEntry(entry $entry, Partner $toPartner = null,
  		
  		KalturaLog::log("copyEntry - New entry [".$newEntry->getId()."] was created");
 
-		if ( $entry->getStatus() != entryStatus::READY ) {
+		if (in_array($entry->getStatus(), [entryStatus::READY, entryStatus::NO_CONTENT]))
+		{
+			self::copyEntryData( $entry, $newEntry, $copyFlavors, $copyCaptions );
+		}
+		else
+		{
```

Comment:
Need to check when is this copy relevant, if it is relevant for status entryStatus::NO_CONTENT we will have regression